pkg/daemon/discover/discover.go

```diff
@@ -148,11 +148,11 @@ func matchUdevEvent(text string, matches, exclusions []string) (bool, error) {
 // Scans `udevadm monitor` output for block sub-system events. Each line of
 // output matching a set of substrings is sent to the provided channel. An event
 // is returned if it passes any matches tests, and passes all exclusion tests.
-func rawUdevBlockMonitor(c chan string, matches, exclusions []string) {
+func rawUdevBlockMonitor(c chan struct{}, matches, exclusions []string) {
 	defer close(c)
 
 	// stdbuf -oL performs line buffered output
-	cmd := exec.Command("stdbuf", "-oL", "udevadm", "monitor", "-u", "-k", "-s", "block")
+	cmd := exec.Command("stdbuf", "-oL", "udevadm", "monitor", "-u", "-s", "block")
```

Comment:
What was the `-k` option? I don't recall.

Reply:
```
 -k, --kernel
           Print the kernel uevents.

 -u, --udev
           Print the udev event after the rule processing.
```
because one block device will produce one kernel event and one udev event.

Reply:
Ok, so you're saying that using both -k and -u was causing duplication, so we just need -u

Reply:
yes @travisn 
please help to confirm whether need to remove one of parameters, thanks. @satoru-takeuchi

Reply:
@microyahoo It's safe to remove this parameter.